README.md

```diff
@@ -17,6 +17,18 @@ is that we're fully open source. Main features:
  - the web panel will be hosted on Heroku
  - objects/files will be stored in Amazon S3 (in the future will be changed to Amazon EC2)
 
+To start application using default port use command line specified below.
+
+```
+$ java -jar target\nerodesk-jar-with-dependencies.jar
```

Comment:
most of us are using Linux, so it would be better to use slash symbol ```/``` instead of backward slash